user.go

```diff
@@ -193,14 +229,32 @@ func (c *Client) CurrentUserFollows(t string, ids ...ID) ([]bool, error) {
 	return result, nil
 }
 
-func (c *Client) modifyFollowers(usertype string, follow bool, ids ...ID) error {
+type userResourceType uint
+
+const (
+	followingResource userResourceType = iota
```

Comment:
This PR looks great, thanks!

The only thing I'd change is these constants. As written, it's easy to make a mistake and forget to keep the resource constant in sync with `userResourceTypeStrings`.

Can we just make the constant a `string` instead of a `uint`?  Then there would be no need for a constant and a slice to lookup the string.

Reply:
Sure thing, I supposed it doesn't need to be strongly typed as it's a private method.